x-pack/legacy/plugins/siem/server/lib/detection_engine/routes/schemas/schemas.ts

```diff
@@ -110,6 +110,18 @@ export const updated_by = Joi.string();
 export const version = Joi.number()
   .integer()
   .min(1);
+export const action_group = Joi.string();
+export const action_id = Joi.string();
+export const action_actionTypeId = Joi.string();
+export const action_params = Joi.object();
+export const action = Joi.object({
+  group: action_group.required(),
+  id: action_id.required(),
+  actionTypeId: action_actionTypeId.required(),
```

Comment:
Same comment above about if we can use snake case here or is that too painful due to the way alerting is structured? snake case would be more consistent if we could. I leave it up to you.